python/pyhdk/hdk.py

```diff
@@ -2094,8 +2094,8 @@ def device_type(self, value):
 
 class HDK:
     def __init__(self, **kwargs):
-        if "debug_logs" in kwargs:
```

Comment:
Why do you need this change? `log_severity` allows you to choose any logging level. `debug_logs` always enables max debug log level.

Reply:
`log_severity` is not captured on initialization and having two options for debug is a bit confusing. I have now changed `debug_logs` to boolean or string so that on `True` we just do max level logging and on `"INFO"`/`"DEBUG1"`/...  we do `log-severity`-like logging. Would it be a good solution?